plugins/Toolbox/resources/qml/pages/ToolboxInstalledPage.qml

```diff
@@ -106,8 +113,106 @@ ScrollView
                 }
                 Repeater
                 {
-                    id: pluginList
+                    id: installedMaterialsList
                     model: toolbox.materialsInstalledModel
+                    delegate: ToolboxInstalledTile { }
+                }
+            }
+            Label
+            {
+                visible: toolbox.materialsInstalledModel.count < 1
+                padding: UM.Theme.getSize("default_margin").width
+                text: catalog.i18nc("@info", "No material has been installed.")
+                font: UM.Theme.getFont("medium")
+                renderType: Text.NativeRendering
+            }
+        }
+
+        Label
+        {
+            anchors
+            {
+                left: parent.left
+                right: parent.right
+                margins: parent.padding
+            }
+            text: catalog.i18nc("@title:tab", "Bundled plugins")
+            color: UM.Theme.getColor("text_medium")
+            font: UM.Theme.getFont("medium")
+            renderType: Text.NativeRendering
+        }
+
+        Rectangle
+        {
+            anchors
+            {
+                left: parent.left
+                right: parent.right
+                margins: parent.padding
+            }
+            id: bundledPlugins
+            color: "transparent"
```

Comment:
A Rectangle that is "transparent" should be an Item

Reply:
Nope, it has a border.

Reply:
My bad